README.md

```diff
@@ -21,7 +21,14 @@ This README documents the current strawman architecture.  Changes must be made v
     2. ARAs respond to ARS with Message
     3. ARA sends query messages to KPs
     4. KPs respond to ARAs with Message
-3. All communication between components conforms to the ReasonerAPI Message spec
+3. Interfaces:
+    1. All communication between the ARS and ARAs conforms to the ReasonerAPI Message spec
+    2. KP can expose their information using these methods:
+       A. ReasonerAPI Message
+       B. Any SmartAPI-annotated interface
+       C. A file dump conforming to KGX standards
+    3. Teams providing SmartAPI annotated interfaces or file dumps will also provide the same information via a ReasonerAPI.
```

Comment:
Again, apologies for missing the call.  Can someone provide the context on this addition?  Seems to me that KPs satisfying any _one_ of the three interfaces would be sufficient.

Reply:
We didn't discuss it much there, so you didn't miss anything in particular.  I added that based on what I picked up from the slack conversations - it sounds like the smartAPI group is standing up this service already, while recognizing that it might generate a bottleneck.    If we need to revisit after gaining experience, we can always do so.

Reply:
Totally fine with revisiting and revising this document later.  But as written, I interpret point 3 to impose an added burden on KP providers. For now, I think KP providers should focus on providing their data via any _one_ of the three stated methods so we can asses whether that is sufficient for data integration.  I just don't want KPs to have to fulfill responsibilities that later may prove to be redundant. Yes, we will provide a BTE instance that will effectively do transformation from SmartAPI to Reasoner Std API, but I don't think that's a responsibility of the KP teams.  (Also, the BTE instance won't address the needs of KP teams who are providing KGX-compliant file dumps.)

Reply:
And without it we put a (different kind of) burden on consumers.  What if we modify the language a bit to something like: "The Translator consortium will develop tools to automatically A) proxy ReasonerAPI calls to smartAPI calls and B) automatically deploy ReasonerAPIs of KGX file dumps".  Then the responsibility is at the consortium level for the document.  In real life, though, I think it will only succeed if the guts of the ReasonablySmart Proxy are BTE based and maintained.   But I could imagine a role for e.g. SRI to work on the actual proxy instance, and/or handle some tools for doing data2services or automat like things with KGX files.

Reply:
Part A satisfies my previous concern (asking KPs to provide their data via more than one Translator-approved methods).  From a consortium perspective, I don't quite understand the rationale of Part B, but since it doesn't directly impact any of our activities I don't want to hold this up any more over that.  So with that change, this PR gets a 👍 from me.

Regarding the "in real life" addendum, I continue to believe that success will involve direct usage of the BTE library by ARAs that want to integrate SmartAPI-based KPs, and not a proxy. But again, this is one of the many things that I am highly supportive of empirical testing.